cmd/ocsp-responder/main.go

```diff
@@ -292,13 +298,13 @@ as generated by Boulder's ceremony command.
 
 		source = &dbSource{dbMap, filter, c.OCSPResponder.Timeout.Duration, logger}
 
-		// Export the MaxDBConns
+		// Export the values for dbSettings
 		dbConnStat := prometheus.NewGauge(prometheus.GaugeOpts{
 			Name: "max_db_connections",
 			Help: "Maximum number of DB connections allowed.",
 		})
 		stats.MustRegister(dbConnStat)
-		dbConnStat.Set(float64(config.DBConfig.MaxDBConns))
+		dbConnStat.Set(float64(config.DBConfig.MaxOpenConns))
```

Comment:
Just since the existing config value is being exported as a stat, any desire to also export the three new ones, too?

Reply:
This was something I actually meant to ask about but it slipped my mind. It seems that it would make a lot more sense to just export the config value in the call to sa.InitDBMetrics up on line `289`. That calls the following code:

https://github.com/letsencrypt/boulder/blob/2d14cfb8d1bc56e70e61f8d50d8f5a150e9b6c1d/sa/metrics.go

Reply:
I can either open another issue or ship another commit to address this.

Reply:
Chatted with @aarongable and decided to just ship another commit into this PR adding it to sa.InitDBMetrics